src/bun.js/bindings/ErrorCode.cpp

```diff
@@ -725,6 +761,11 @@ JSC_DEFINE_HOST_FUNCTION(jsFunction_ERR_UNHANDLED_ERROR, (JSC::JSGlobalObject *
     return JSC::JSValue::encode(createError(globalObject, ErrorCode::ERR_UNHANDLED_ERROR, makeString("Unhandled error. ("_s, err_str, ")"_s)));
 }
 
+JSC_DEFINE_HOST_FUNCTION(jsFunction_ERR_IPC_ONE_PIPE, (JSC::JSGlobalObject * globalObject, JSC::CallFrame* callFrame))
```

Comment:
Instead of adding more functions, modify the one at the bottom to throw this message by default when `$ERR_IPC_ONE_PIPE,` is passed through JS.